notebooks/406-3D-pose-estimation-webcam/Visual3D/main.ipynb

```diff
@@ -0,0 +1,82 @@
+{
+ "cells": [
```

Comment:
what's the propose of this notebook ?  Could you help to add a README file or explain more in this notebook ?